### PR TITLE
coreos/azure: Updates for v1.2

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

**Comment by errordeveloper** [comment]
_xref https://github.com/kubernetes/kubernetes/pull/23818_

**Comment by errordeveloper** [comment]
@johndmulhausen ping

File: docs/getting-started-guides/coreos/azure/index.md [hunk 1]
```diff
@@ -6,225 +6,224 @@
...
+data:	   Virtual IP Address		 : 137.117.156.164
+data:	   Direct server return 	 : Disabled
+info:	 vm endpoint show command OK
+```
+
+You then should be able to access it from anywhere via the Azure virtual IP for `kube-00` displayed above, i.e. `http://137.117.156.164/` in my case.
+
+## Next steps
+
+You now have a full-blow cluster running in Azure, congrats!
```
Comment (johndmulhausen) [inline docs/getting-started-guides/coreos/azure/index.md]: How about: full-blown Kubernetes cluster

**Comment by johndmulhausen** [comment]
Nicely done/written! No complaints here except for the nit. Ready for a tech review.

**Comment by errordeveloper** [comment]
@johndmulhausen I've address the typo you pointed at :)

**Comment by errordeveloper** [comment]
This is clean-up commit, does anyone have concerns about removing this or moving it into a separate repo?

**Comment by errordeveloper** [comment]
@johndmulhausen ping.

**Comment by errordeveloper** [comment]
I've rebase this. The diff is still a little obscured by EOL changes, but you can look at it here https://github.com/kubernetes/kubernetes.github.io/pull/314/commits/572bd36791b501816deb857a9ead7cd28b405bb6?w=1

**Comment by errordeveloper** [comment]
The kubernetes/kubernetes#23818 is stalled, it'd be great to get that merged - it's just a clean-up.

**Comment by errordeveloper** [comment]
@pwittrock yes please, and https://github.com/kubernetes/kubernetes/pull/23818.

**Comment by pwittrock** [comment]
@errordeveloper SGTM.  Would you rebase and I will merge?

**Comment by errordeveloper** [comment]
@pwittrock rebased.

**Comment by kelseyhightower** [comment]
@errordeveloper LGTM.

**Pull request merged**